### DISCUSSION TITLE
Change border color to all special workspaces

### DISCUSSION THREAD

**Discussion by sickmitch** [hyprwm/Hyprland#2889] [Q&A]
I've been fiddling around for quite a bit trying to get a window rule to work on all special workspaces to change the active border color of all windows in them.
I've trying defining a rule for all special workspaces using workspace= but this don't receive regex.
With a window rule like this 
`windowrulev2 = bordercolor rgb(FFFF00),name:^(special.*)$` 
I only found this in the wiki window rules section but i can't quite understand how to write it down.
`workspace - id or name: and name`
With hyprctl clients i get that : the only common thing in the windows i want to change is the 2nd field in the workspace line which starts with special.
The first field ig is the ID and change from boot to boot.

**Comment by Scelester** [comment]
Any update i am stuck on this

**Reply by enthusiva** [reply]
windowrulev2 = bordercolor rgb(f5c2e4), onworkspace:s[1]

**Comment by ChristianFredrikJohnsen** [comment]
windowrulev2 is now deprecated. Instead use:
`windowrule = border_color rgb(f5c2e4), match:workspace s[1]`
